uploader.json

```diff
@@ -6307,6 +6312,11 @@
                     "archive_file_name": "/insights_commands/ss_-tulpn",
                     "command": "/usr/sbin/ss -tulpn",
                     "pattern": []
+                },
```

Comment:
No need to update the 'Specs' part.

Reply:
"/insights_commands/ss_-tupna" Source file location will be required,  right?

Reply:
Yes, it can be done by the framework
Btw, `uploader.v2.json` should apply the same update.

Reply:
Done, removed